### DISCUSSION TITLE
Can we address the Verified Tick?

### DISCUSSION THREAD

**Discussion by addanus** [deso-protocol/dips#49] [Large CIPs]
💯 Many committed Bitclouters have been left out (or chose to stay out) of other socials. They (we) are all-in for Bitclout and spend a ton of time focused on various Bitclout activities. We absolutely need a process to identify and highlight them - over transplants from other socials who often aren't even engaged here.  👈

🤔 Verification currently occurs per node and is therefore not shared. Plus the authentication process is opaque, so I'm not sure it's a good idea to blanket-accept verified status from other nodes.

😠 Apparently scammers have either used fake verified twitter accounts or coerced verified accounts to post in order to rugpull.

👏 I've pointed it out elsewhere, but everything the core team has done to growth-hack bitclout to this stage can be applauded (or forgiven) - including incentivizing 15K twitter accounts with real $. However, now that we're here, we need to support the community that's invested immeasurable amounts of sweat, tears and body real estate.

**Comment by addanus** [comment]
From a discussion on discord:

First we should clarify/change what the mark means. Maybe it should be 'real human creator/dev whose actively making bitclout better' - then apply checkmarks accordingly (and remove it from those already so marked).

On the trust side, we can do a similar thing for nodes - accepting certain levels of transactions from certain nodes. If we can agree on verification processes, we can then allow certain trusted nodes to do it - and have that propagate to the rest of the nodes. 

We're essentially duplicating DNS or other similar internet protocols - just applying it to social... in case that wasn't already clear.

**Reply by marnimelrose** [reply]
On our node for instance, the verified people are people who are coaches, mentors, trainers and freelancers who we have interviewed. Not sure if that will always be the case but it is our use case at the moment.

**Reply by addanus** [reply]
Great point - and suggests that maybe we should keep it as a node-level feature.

In that case, we would consider what's important on bitclout.com itself. Does the community value twitter accounts above bitclout contributors?

It would also help to know if the near-future bitclout.com will surface/highlight other nodes instead of individual accounts - verified or not.

**Reply by tijno** [reply]
Worth noting this from the [FAQ](https://docs.bitclout.com/faq/bitclout-faq#what-parts-of-bitclout-are-centralized-at-bitclout-com-and-what-parts-are-decentralized-on-the-bitclout-blockchain-and-what-parts-of-the-bitclout-blockchain-are-still-semi-centralized):

> ... Instead, the core dev team has plans to introduce an “association” primitive that will allow profiles to associate with one another, effectively creating a more decentralized version of the blue checkmark. For example, your profile can list associations with universities or workplaces that have been cryptographically signed.

**Reply by addanus** [reply]
Appreciated. This would be a neat use case (sort of like linkedin - but with validation from the other side). 

We still have the problem of the "community" bitclout.com recognizing notable/verified creators/devs/contributors - beyond just the few who've gotten the golden blessing from core team.

That's why it would help to update/codify what the mark means (via CIP voting) and then proceed to bestow the privilege fairly. Recall the huge problem is you have to be verified on an unrelated (and often hostile) platform to get that here.

**Reply by sull** [reply]
> Worth noting this from the [FAQ](https://docs.bitclout.com/faq/bitclout-faq#what-parts-of-bitclout-are-centralized-at-bitclout-com-and-what-parts-are-decentralized-on-the-bitclout-blockchain-and-what-parts-of-the-bitclout-blockchain-are-still-semi-centralized):
> 
> > ... Instead, the core dev team has plans to introduce an “association” primitive that will allow profiles to associate with one another, effectively creating a more decentralized version of the blue checkmark. For example, your profile can list associations with universities or workplaces that have been cryptographically signed.

I have a feeling that associations won't happen by the core dev team, although they may provide some tooling for it in the node software but it would still require node operators to do the actual associations either manually or via private APIs (e.g. University Attended, Place of Employment, KYC etc). 

In other words, we probably won't see it in action on bitclout.com since they have plans to shut down that node and repurpose the site (assuming) to be general informational resource (docs etc) and maybe provide a list of "trusted" nodes.

**Reply by sull** [reply]
> We're essentially duplicating DNS or other similar internet protocols - just applying it to social... in case that wasn't already clear.

It would be interesting and some nodes may interop in such ways (web or trust). 

There probably should be two or more layers of verification for nodes to leverage.

1. Node-specific like @marnimelrose use case
2. Node-specific for general connected account associations (e.g. Prove control of a Twitter, Linkedin etc)
3. Nodes that utilize "associations" primitives (likely need to form relationships/partnerships with 3rd party entities)
4. Node trust network for verification inheritance (relates to all other verification types)

**Reply by dgsus** [reply]
While I do believe this might be temporary, due to associations, I agree that the current community growing bitclout's ecosystem  - either with code or ideas - should be rewarded with the badge. Which most likely will be just that in the future, if bitclout dot com is decommissioned.

**Reply by leeceedee** [reply]
Is there any particular reason it has to be just 1 type of verified tick? Could there be a number of them instead? verified by node, verified because famous, verified as continuous bitclout member/contributor, verified as pre-bomb account etc etc. Diff colour for each tick. Then it could sort of be like the old geek code; the more verified ticks for xyz reasons, the more complete the description of the person behind the account.

**Comment by tijno** [comment]
Not to hijack the valid discussion - so posting this as a separate thread here.

A related challenge is that some previously reserved accounts that are now verified, still show up as "reserved" on nodes - even though they are active.

I wish there was a way to have one way sync to my node - so it copies status from bitclout.com to mine for any verified accounts.

Node:

![20210712-gIyNn5U8](https://user-images.githubusercontent.com/69529928/125346011-23f44800-e351-11eb-83fd-16ac2c47942c.png)

Bitclout:

![20210712-SMEl4hjt](https://user-images.githubusercontent.com/69529928/125346019-2787cf00-e351-11eb-9ebb-c287b6c499fb.png)

**Reply by addanus** [reply]
Totally. At least have an option to the point that @marnimelrose mentioned.

Some nodes might not want to pull verified status from the main nodes - or might want to start all accounts as unverified and build up from there.

I really think we can do this like how DNS works. On my laptop (or home internet network) I can choose to accept the domain address for any website or set it how I want. On bitclout that domain is just the public key to any account.

**Reply by dgsus** [reply]
Maybe bitclout dev team could provide a weekly export, or such, of their verified db? 

Also, it would be awesome if the verified/whitelist/blacklist etc allowed for multiple input separated by commas. If I'm not mistaken it only accepts single entries.

**Reply by sull** [reply]
> Maybe bitclout dev team could provide a weekly export, or such, of their verified db?

but that might not scale since we do not know for how long they would be a reliable source, especially once they take down their node (if this remains the plan).

**Reply by dgsus** [reply]
Indeed, a simple/temp solution for the wishes of @Tijno

For the long run, perhaps a kind of top level service / dao could provide verification as a service for node operators?

**Reply by 0xNordian** [reply]
> Also, it would be awesome if the verified/whitelist/blacklist etc allowed for multiple input separated by commas. If I'm not mistaken it only accepts single entries.

Agree with this!

**Comment by FreeTrade** [comment]
Ideally what we'd want is to have all the verification information available on-chain, and then collate and interpret that information in a way that is specific to a user. So each verifier is a participant in a web-of-trust type system, but their inputs are only weighted if they are trusted by the evaluator. This is a decentralized way to approach it.

**Reply by HPaulson** [reply]
A. How would these txs be signed? Nodes don't have a keypair

B. Bloat

**Reply by FreeTrade** [reply]
Just saw this - so whoever is verifying does the signing and submits the transaction. Would increase onchain usage, true, but not much more than some follows. If each user was verifying even just 20 other users, you'd already have a very informative web-of-trust.

**Reply by FreeTrade** [reply]
I'd encourage everyone to review the work we've been doing on this -
1. member.cash has a membrain score, which ranks members on a 1 to 5 scale, trying to assess how real and valuable a member is
2. we parse @verifiedprofile posts, so this has become the first verification provider
3. all information is provided in a trust graph, that is personalized to the user - more details on that at
https://member.cash/p/fb5c7342d9
and
https://member.cash/p/bd9883af5b

**Comment by omicreativedev** [comment]
Coming from a personal place here, I've requested verification by email and never got a response that I'm aware of- not even a no. I'm only verified on Apple, Spotify, etc. not Twitter, Insta, etc. I got my check removed for TOS violation (they changed their TOS, smh, wasn't even my fault smh.)

For myself, much like a porn, "Explicit Music" and battle rap is generally against most mainstream social media TOS now. It might be provocative or contain language not suitable for kids. In a way, we don't wish to dissent from social media. We are made INVALID. 

Moreover, the very things that determine verification on social media today is aimed at cherry picking those that do fit the status quo kid friendly mainstream pop media— unless you're an uber-celeb, then the rules don't apply to you.

And yet lots of sites just default to using their verifications (such as Audius, smh) and don't even know what goes into their verification process. Instagram and Twitter verify based on things like having no strikes against their (very uptight, very centralized, very non-free speech) TOS, having imposters, number of google search results, being in a vetted library of press (like Forbes or Billboard), giving up your ID, phone number, etc. And it _doesn't_ really work. There are lots of imposters and posers on both platforms, and accounts that aren't even people; who know how to beat their system- like paying to be featured in press and using SEO techniques and ads to bump their site rank. Its just forgiven because it's 75% effective.

The whole system is set up to BIAS FOR commercial mainstream pop youth-centric media tho, unless of course you're an undeniable multimillionaire or have a million followers- then by all means, be as vulgar as you like. Its based on their TOS and not things like "free speech" "free trade" "decentralization" ACTUAL validation of users, etc.

For example, YOUTUBE demonetized and unverified a whole genre of entertainment, Battle Rap. Except for several really large leagues. So most have gone to Caffeine who will verify them and treat them as valued content creators. It's sad most of them wouldn't be able to get verified on Bitclout, Or only a node where others are verified who aren't battle rappers. See the problem for future building?

It's these dissenters, deplatformed, de-validated, folks that seek out alternatives most too. If I had a Battle Rap node, it would easily be 100K new people. They are hella active users and fans too.

Thank goodness for Spotify that verified me and that keeps on verifying all musicians by their actual credentials. When I lost my Twitter check, I lost my Facebook check too. Smh. Why? Sharing videos with explicit lyrics and violence. But only small accounts are punished- never Cardi B who can just post anything. The system is corrupt.

I digress, we are copying a corrupt system that is the opposite of free.

**- I believe that Verification should be NODE BY NODE. Let companies and projects decide who is verified in their establishment. Simply have a switch for invoking main node verification list/update if desired. Otherwise, start blank. I've heard several independent node owners complain that validated accounts show up on their node. What if they want their node to be a company and verify only employees? What if it's a club?**

- As far as Main node verification, I know it's probably faster to just lean on twitter and it goes along with the whole reserve accounts thing but I feel like Bitclout should consider hiring or having a community person on that list as it's really important to people. It might even be considered that if an account is verified on 2/3 of nodes, bitclout.com will verify. Or even just broadening the verification to INCLUDE fair verified systems like Apple and Spotify for artists and Linkedin for professionals.

As for community contributors, developers, etc. of course I would vote for yes. Duh, I want a check! But ultimately, I already have one on a bunch of nodes and all my own nodes so no big deal.

**Reply by AikonMishima** [reply]
I wholeheartedly agree, especially with the verification tick being given at the node-level.

There’s a variety of cases where a blank slate will be required:
- Specialists community (love4src’s for instance, only gives the tick to developers)
- Coin Holders (only give the tick to coin holders of the node hoster’s Creator Coin)
- Celebs only
- etc

**Comment by omicreativedev** [comment]
Also- while inappropriate maybe, hey @tijno can I get a check on your node since my front end runs on it lol! <3

**Reply by tijno** [reply]
right now the only one verified on tijn.club is me. Im not adding manual verification at the moment - as its not as easy as verifying someone - it needs a more reliable solution. Interested on what that might look like - if you have ideas do ping me on discord.

**Comment by trevormil** [comment]
Wanted to chime in here with a potential solution for the implementation of this. So, I have been working on building BitBadges which offers a solution where users can "associate" with other BitClout users through a digital NFT badge that can't be sold ever (linked to public key). We currently store our data on IPFS and post the hashes of the IDs on the BitClout chain via @BitBadgesHash. With our solution, anyone can issue verification NFTs to whichever other users they want to. Users and even nodes can then choose which verification system they would like. Just have it in settings like say use bitclout.com verification badges or use mynode.com verification badges. TThis way we create a directed graph of who each user trusts and decentralizes the verification process. If you don't like someone's verification process, switch to another user's. If your node has a niche and only wants to verify your clients for example, switch to your verification process. If you have any questions about implementation of BitBadges, I'd be happy to help.

**Reply by trevormil** [reply]
Check out https://bitclout.com/posts/ef35f86874b530a47d969129b599ff08960c8cc6a7d91a8bd410e64f0d42a448 if you need a little more info on @BItBadges

**Comment by sull** [comment]
That’s what badges are useful for. But color coded checks could be useful
too.

**Comment by HPaulson** [comment]
People should just stop caring about the verification badge outside of projects (For projects it matters because favoritism). The verification tick was created (originally) as a method to reduce impersonation for individuals whom this was a large problem -- not a status symbol for people who are active on a platform. 

I should also note this is coming from someone who is verified, so I offer a unique perspective on such because honestly it *doesn't matter*. Would give 0 fucks if I had my tick removed, it's not supposed to be a status symbol, so please let's stop making it one. Just my two cents

**Reply by sull** [reply]
You're not wrong.

**Reply by leeceedee** [reply]
Has anyone looked into coin value pre & post verification? I think verification matters more on bitclout not because it's a status symbol, but because it's a message that says 'this person is x% more trustworthy' compared to bot accounts or fake accounts etc, and therefore ppl are more comfortable investing in them.  It adds value to the account itself and removes a potential (fear) barrier for cautious investors.

**Reply by sull** [reply]
maybe, idk. 

If a basic verification was tied to whether or not an account could change FR from 100%, that could be interesting.
Basically the opposite of what the dev team launched with (terrible approach with 10% for all accounts).

**Reply by DiscoGroup** [reply]
Okay then change your username @HPaulson and set an example that verifications doesn't matter....

**Comment by audionode** [comment]
From a node owner's perspective, I can't imagine allowing cross-node verification unfortunately.  At AudioNode, we don't even accept BitClout verified artists as our internal stamp of approval.  We just show users who have been verified by us, who by BitClout, and who hasn't been verified using either, and let users make their own decisions about which process to trust.  I could see in the future showing other node verifications as those nodes open APIs for us to compile that data, but outside of that I simply don't see a path to any kind of universal verification.

Due to the liability surrounding verifying accounts and sending a signal to traders that an account has been approved I simply can't imagine nodes (which are legally liable C-corps ... AKA, not decentralized and mostly immune coin-funded "entities") ever accepting other node verifications as their own (without tight contractually binding agreements).

It is really a legal hurdle more than anything else.  Solving it with tech is the easy part, solving it legally seems close to impossible.

**Reply by tijno** [reply]
Very valid point - if verification is not an objective transactions that can be verified - it could introduce legal risk for sure.

**Comment by omicreativedev** [comment]
I would like to present evidence that it is not all in our heads and a status thirst thing. Verification means something to audience in a very MENTALLY IMPRINTED WAY for trust and respect and worth. Respectfully, you don't have to agree for it to be true.

Here is a screenshot (one of several) where directly in my case, I was directly pushed away due to lack of verification. And there are many others. Ie trust/respect. As far as worth, it's obvious verified accounts get more investments and more of certain investors attention. I'm sure many others have had similar experiences. I've also been impersonated on Bitclout to the point I've bought coins of fake-me's just so people can find a trail back.

It is a barrier some people won't understand because they either have the verification or haven't experienced direct prejudice without it. Sort of like a ethical blindspot. "It shouldn't be important" isn't a fair argument- it's a moral judgement. [And I say that respectfully disagreeing.]

It IS actually what it is, and it does what it does, like it or not in the minds of audience. That's truth. (Don't shoot the messengers.)

![UGH](https://user-images.githubusercontent.com/82254764/125510677-9d063cb1-fbca-43fa-89d0-a132880f71b5.jpg)

So I think it's a good idea to discuss and for everyone to have empathy for those who aren't as fortunate in this respect. 

Lets say I onboard 10k people?
Lets say I invest 10M dollars?
Lets say I add 1000 hours of dev work?
lets say I give you my actual passport?
At this point it wouldn't matter.
Because Twitter (the competition?) is who is really validating _our_ users.

I say this all with deepest respect and love. I feel this is a fair open honest discourse.

**Reply by DiscoGroup** [reply]
@teambrixmusic Is right , the verification on bitclout is an absolute joke. I have got several accounts(30+) verified legitmently and illegitmently. I have sent emails to devs on outlining issues, zero response is NOT acceptable when your dealing with peoples $$. I have brought celebs on the site and within 60 days they have all left, this core team is really out of touch with how to retain users. From this thread it sounds like they are starting to lose their biggest advocates, myself included.

**Reply by tijno** [reply]
Agree verification status matters to majority even if only for the first split second in making the decision to pay attention to someone or someones post.

Bitclout.com approach is [outlined here](https://docs.bitclout.com/faq/bitclout-faq#what-criteria-are-or-will-be-used-for-user-verification-as-it-seems-inconsistent-at-times-and-will-there-be-away-to-get-verified-without-social-profiles-itsaditya-lukasjakson)

> Today, bitclout.com verifies users if they have a verified checkmark on Twitter, with very rare exceptions in cases where it looks like a user will add a lot of value to the community. 

I dont use twitter or IG so am not verified there. I got verified here I guess as an exception due to value added - which i was happy with of course.

Note that it did not affect my coinprice as far as i can see.

**Comment by kanshi** [comment]
Please consider a following set of solutions, as a replacement of the current verification system.

Stakeholders: creators, traders, communicators, (infra/nodes/apps) operators, scammers

Concerns / goalz:
1. Having a systematic, commonly understood and accepted approach to demonstrate the peer group of an account
2. Having an ability to benchmark accounts in the context / area of interest / perspective of the observer
3. Having a distributed, flexible model, ideally mapped to (nodes/apps/external) communities
4. Having a common language when we speak about account reputation/verification/affiliation/association
5. Avoiding complexity explosion and bloating the blockchain or the databases
6. Not enforcing why people should care about others, but giving them tools to quickly figure it out

Discussed models included:
* standardized approach to interpret the current followers graph as a network of recommendation, graded on degrees of separation (compute intensive, hard to cache personal preferences)
* defining a function (karma-alike?) to categorize recent activity:
  - creators - posts weighed by meaningful activity, nfts, clout, nodes presence, ...
  - traders - timespans, ROI, quality of recommendations, scope of activity, clout, nodes presence, ...
  - communicators - reach, magnitude and effect of posts, reclouts, quotes, nodes presence, ...
  - operators - capacity, impact, uptime, clout, growth, ...
  - scammers - impact, clout, methods, ...
* a badge system that would consider activity from a recent timespan
* github/gitlab heatmaps of activity with different colors representing activity kinds (vulnerable to bots, accessibility concerns)
* color based representation of the recent activity (hard to quantify unified color representation, accessibility concerns)
* Flame by kanshi from 1st BitClout Producthaton (noticeable learning-curve, hard to master)

Thank you teambixmusic for pinging me, thank you all above for great comments that got me into this discussion, thank you paulmp, znmead michaelgiordano theparkeraziii slava  for amazing brainstorming session at the nachoaveragecall ❤️ 

Happy to consider this further 👍

**Reply by tijno** [reply]
Theres some amazing stuff here! But probably covering several CIPs :)

**Reply by kanshi** [reply]
I'm personally siding with a "live badge system" where we could discuss balancing out the timing + blockchain activity to map it to the badges (as a convention) displayed on the profile. Seriously considering testrunning this ...

**Reply by bloated-devish** [reply]
> * Flame by kanshi from 1st BitClout Producthaton (noticeable learning-curve, hard to master)

can you explain this a bit more?

**Comment by sidikinoel** [comment]
@HPaulson is right on about verification on most social media platforms being aimed at resolving impersonation, and over time we've all seen the coveted blue check turned into some form of status symbol.  

Whether one cares or not about the blue check is an individual preference/need, however, unlike other platforms, the creator coin economy on BitClout and nodes of the like have expanded the problem beyond just impersonation in my opinion.  There now is a problem of trust.  

If we set aside for a moment the argument that the blue check has been comprised in more than one occasion by scammers, on BitClout trust is indicated by the BlueCheck.  So everyone else who is NOT a BlueCheck is in the same bucket with scammers, bots, etc... 

How can a user effectively, reliably and easily assess trust when interacting with a profile and convey trust in their profile?  And is that process simple, scalable and portable **if needed** as users interact across the eco-system of nodes? 

That is why another indicator, check color, something is needed to provide users some level of confidence that some due diligence was done by the node administrators in providing recommendations on profile veracity.

As you think through all of the issues @addanus brought up: 

- opaque verification process 
- node centric verification process
- scams
- leveraging compromised sources of truth

they all boil down to a problem of trust at multiple/various levels.

I cannot envision a decentralized process where verification from each node operator are ingested by/shared out to other nodes without a way to audit individual processes for trust and continually monitor the process quality over time (Audionode's comment is case in point), or without a way to ensure that all contributing nodes are adhering to the same verification standard.

If each node operator hands out their respective badge or check for verification, using differing processes, we'd run into not only the issue aforementioned but also usability issues for the user base, and would create auditing nightmares.  Having to get verified every time one jumps on a new services/node despite being verified on another isn't ideal.  As a user, i also don't want to have to personally vet the verification process of every service/node i jump on.  I'd love to do this once and then knowing that the services I use adhere to the verification standard I have vetted, trust and understand goes a long way.

BitClout hands out blue checks, @cvl we hand out an orange check, other services hand out an NFT badge, others have spoken of a grey check.  Why should anyone trust the blue check, orange check, the grey check or badge without being able to audit the verification process.

I also don't believe verification only can address the issue of scams.  Trust is a layered Swiss cheese problem and verification represents a layer.  Even with trusted checks or badges rug pulls can and will happen.  Additional safeguards related to the handling of creator coins are necessary as another layer of defense in enabling trust (This is a whole other topic with lots of exciting ideas/possibilities).

At a high level, the solutions I currently envision for verification are: 
- **Verification Standards:**  It would be transparent and auditable, and employed by nodes that can eventually share their verification ticks because they are all using the same due diligence standard and trust representation.
- **Centralized Verification:** Single point of truth all nodes can pull from but also offload the liability to.  This has the advantages of a standard with the added liability waiver.
 
This is just the what. The how is a whole other can of worms.

These approach can bring about transparency to the process, solve node centric verification issues, establish a viable source of truth, and provide a foundation to build on for defending against scams.  It also provides a user friendly approach that is portable and scalable.

**Comment by bloated-devish** [comment]
I actually like the overall plan that was put forth in other threads that we could have a "relationship" link between nodes and users (profiles) as an edge, with attributes that make it clear whether the user is blocked by the node or verified etc. This would be great and I want to extend this - I want it to be available for all transactions. 

What does this mean? Nodes can "confirm" they created the transaction (post/creator coin transfer / creator coin / nft / whatever). In addition to the edge being added, I'd like the nodes to be able to add attributes to these edges. This allows some extra data to be attributed to the edge, so if we find some spurious node related data, they can all be tracked. Additionally, if we find nodes that are too spammy on the chain, this can also be clearly tracked.

(For those from Graph theory: node here is bitclout.com, cloutfeed etc not the graph node).

## What could be the future of ```ExtraData```
Today there are already some nodes (like member.cash) adding "block user" data to the chain using Extradata. While actually love this change, I think over time this is going to blow up and inevitably, two nodes WILL write the same field with different values and it is going to be a mess.

## Benefits of clear attribution
Clear node "attribution" of data will reduce the scope of bloat issue and control the problem whenever they occur. Even if am the most responsible node operator on bitclout platform, if I have to canary a new feature, it would be hard for me to see what additional bloat I am creating if I add ```ExtraData```, but with the set up I am proposing, nodes can build a dashboard themselves and see how their canary binary is affecting the data in the chain.

## Soft-fork reduction and better treatment
Also this can reduce the scope of soft-fork. When a field becomes widely used and needs to be promoted to chain - We could build a "backend" layer API that loads both this node <-> txn extra data and map it to its field in the transaction when it is "promoted" to its own field. When this happens, the backend API code will be a waterfall like this:
1) If its present in the chain as a first class attribute use it.
2) When its present in the "node's" edge attribute use it.
3) If the field is in default node (bitclout.com) use it.
4) Use default value when the field is missing.

(step 3 above would be removed when we shutdown bitclout.com). Frontends / Nodes shoudl be able to read the field as if its there and we can handle this in the middle layer.

## Automation of ExtraData fields promoted to first class fields
Infact, we can build a script that finds fields that are widely used and provides them as suggestions if there are bitclout committee meetings on deciding changes to consider fields for promotion for soft-fork. This would just bubble up fields, providing insight into which node(s) uses them, what is their average size, what is the standard deviation of the payload (are two nodes using this very differently?).

@maebeam @diamondhands0 @lazynina @redpartyhat @bluepartyhat FYI
